core/src/test/java/org/elasticsearch/index/query/AbstractQueryTestCase.java

```diff
@@ -158,11 +157,6 @@
     private static int queryNameId = 0;
     private static SearchModule searchModule;
 
-
-    protected static QueryShardContext queryShardContext() {
-        return queryShardContext;
```

Comment:
okay, so I understand you correctly: should I revert 4e77adf38 instead and we rather use the instance member as much as possible and try to remove the `createShardContext()` that makes a copy instead?